package.json

```diff
@@ -33,7 +33,9 @@
     "@semantic-release/git": "^10.0.1",
```

Comment:
Need to add `packageManager` and `engines` for proper pnpm with corepack.

Reply:
Currently haven't updated to pnpm on swimm-ui @talkor WDYT